process/templates/WG_NAME_graduation_stage.md

```diff
@@ -1,11 +1,36 @@
 ## Working Group graduation application
 
+### WG has met all Incubating requirements
+  * "link to Incubating PR"
+
 ### List of regular contributors
 The WG must have at least 5 contributors from at least 3 different organizations attending regularly as recorded in meeting minutes.
-
   * "name, affiliation, GitHub ID"
 
-### Working Group activity
-The WG must have met at least 4 times over a period of at least 2 months since becoming `Incubating`
+### Release cadence
```

Comment:
What does "Release" mean in the context of WGs?

Reply:
the WG should have some activities and artifacts that are shared publicly from time to time.  Publishing guides, whitepapers, blogs, something that furthers our missions and goals.